src/models/event-timeline-set.js

```diff
@@ -187,7 +187,7 @@ EventTimelineSet.prototype.resetLiveTimeline = function(backPaginationToken, flu
     newTimeline.setPaginationToken(backPaginationToken, EventTimeline.BACKWARDS);
 
     this._liveTimeline = newTimeline;
-    this.emit("Room.timelineReset", this.room, this);
+    this.emit("Room.timelineReset", this.room, this, flush);
```

Comment:
any chance we can improve the name of `flush` (`resetAll` seems fine) and clarify what it means to flush the non-live timelines?